### PR TITLE
Undo withdraw submission

### PR DESCRIPTION
**Pull request by purujitgoyal**

### CODE REVIEW

File: openreview/conference/templates/withdraw_process.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-def process(client, note, invitation):
+def process(client, note, invitation, existing_note):
```
Comment (melisabok) [inline openreview/conference/templates/withdraw_process.py:1]: don't you need to change the signature to `process_update` here? so the process function is being called when the note is edited or deleted.

Comment (purujitgoyal) [reply openreview/conference/templates/withdraw_process.py]: I did that but it's failing with `InvitationExpiredError` now.

**Comment by melisabok** [comment]
The error in the test is saying the Withdraw invitation has expired, that means you can not delete the note. 

This is invitation is expired when the paper is withdrawn, could you try to remove 'Withdraw' from the regex here: https://github.com/openreview/openreview-py/blob/master/openreview/conference/templates/withdraw_process.py#L71?

**Comment by melisabok** [comment]
The changes look good! Could you verify in the test the message sent to the reviewers/ACs when the submission is restored? 

I'm not sure the text would like right.

**Review by melisabok — APPROVED** [review]

**Pull request merged**